client/sites/components/sites-dashboard-banners-manager.tsx

```diff
@@ -19,6 +20,7 @@ const SitesDashboardBannersManager = ( {
 		useMigrationPendingSitesBanner( { sitesStatuses } ),
 		useDashboardSurveyBanner(),
 		useA8CForAgenciesSitesBanner( { sitesCount } ),
+		useDashboardOptInBanner(),
```

Comment:
Should we change the order as only the first one that should show will be render 🤔